contrib/builder/deb/amd64/generate.sh

```diff
@@ -42,11 +42,19 @@ for version in "${versions[@]}"; do
 	echo >> "$version/Dockerfile"
 
 	if [ "$distro" = "debian" ]; then
-		cat >> "$version/Dockerfile" <<-'EOF'
-			# allow replacing httpredir mirror
-			ARG APT_MIRROR=httpredir.debian.org
-			RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
-		EOF
+		if [ "$suite" = "stretch" ]; then
+			cat >> "$version/Dockerfile" <<-'EOF'
+				# allow replacing deb mirror
+				ARG APT_MIRROR=deb.debian.org
+				RUN sed -i s/deb.debian.org/$APT_MIRROR/g /etc/apt/sources.list
```

Comment:
This doesn't work. I'm not sure how to express this with sed, @tianon may have an idea.

Otherwise we could invoke sed twice for both cases or at least only set the prefix into a variable to avoid duplicating the rest of the code.